src/components/main-menu/MainMenu.test.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import cases from "jest-in-case";
+import { Router } from "react-router-dom";
+import { createMemoryHistory } from "history";
+import MainMenu from "./MainMenu";
+
+test("should display title", () => {
+  render(<MainMenu />);
+
+  expect(screen.getByText(/atom/i)).toBeInTheDocument();
```

Comment:
```suggestion
  expect(screen.getByText(/atom/i)).toBeVisible();
```

Just throwing this in here so I can do it from the web UI